tools/astarte_e2e/lib/astarte_e2e/service_notifier/email.ex

```diff
@@ -35,9 +36,11 @@ defmodule AstarteE2E.ServiceNotifier.Email do
     |> text_body(text)
   end
 
-  def service_up_email do
+  def service_up_email(failure_id) do
     text = """
     AstarteE2E service is back to its normal state at #{current_timestamp()}.
+
+    Last FailureId: #{failure_id}
```

Comment:
I would put "Linked FailureId", "Referenced FailureId" or something like that, something that expresses the fact that this was linked to the down email